app/controllers/projects_controller.rb

```diff
@@ -4,11 +4,15 @@ class ProjectsController < ApplicationController
     before_action :authenticate_user!
 
     def index
-        if (current_user.role=="manager")
-            
-            @projects=current_user.created_projects
+        if (current_user.role == "manager")
+            @projects = current_user.created_projects.paginate(page: params[:page], per_page: 3)
```

Comment:
@Usmanksri Please use eager loading here in the view you are fetching members I guess.

Reply:
SELECT "projects".* FROM "projects" WHERE "projects"."creator_id" = ?  [["creator_id", 1]]
I think here we don't need of eager loading because it running only one query from one table.